modules/javafx.graphics/src/main/java/javafx/geometry/Point3D.java

```diff
@@ -410,10 +410,14 @@ public Point3D interpolate(Point3D endValue, double t) {
     }
 
     /**
-     * Returns a hash code value for the point.
-     * @return a hash code value for the point.
+     * Determines whether or not two objects are equal. Two instances of {@code Point3D}
+     * are equal if the values of their x, y, and z properties are equal.
+     * @param obj an object to be compared with this {@code Point3D}.
+     * @return true if the object to be compared is an instance of Point3D and
+     * has the same values; false otherwise.
```

Comment:
`true`, `Point3D` and `false` should be in `{@code}`.

Reply:
...correction, found in toString() as well.

Reply:
I think that we should explain what makes 2 objects equal. Even if we don't explicitly name the methods used for comparison, we could say "2 points are equals if their coordinates are equal".

By the way, I have https://bugs.openjdk.org/browse/JDK-8226930 assigned to go over some dubious equals/hashcode implementations, in case you want to delegate the task.

Reply:
Agreed, mentioning equality criteria sounds good to me too. How does this look ?

```
    /**
     * Indicates whether some other object is "equal to" this one.
     * Two instances of Point3D are equal if the return values of their
     * {@code getX}, {@code getY}, and {@code getZ} methods are equal.
     *
     * @param obj the reference object with which to compare
     * @return true if this Point3D is the same as the obj argument; false otherwise
     */

```

Reply:
This is the current proposal by Douglas and align with `Object::equals` which it is overriding. I'm fine with this wording. Just need to add the `@code` tags.

Reply:
I have added a commit to match the proposed block by arapte.